src/components/Portfolio/LoanFilterBar.vue

```diff
@@ -55,18 +57,45 @@
 </template>
 
 <script setup>
-import { ref } from 'vue';
+import { ref, inject } from 'vue';
 import { KvSelect, KvTextInput, KvButton } from '@kiva/kv-components';
+import logReadQueryError from '#src/util/logReadQueryError';
+import userIdQuery from '#src/graphql/query/userId.graphql';
 
-defineProps({
+const props = defineProps({
 	totalLoans: {
 		type: Number,
 		default: 0
 	}
 });
 
+const apollo = inject('apollo');
 const searchText = ref('');
 const selectedStatus = ref('all');
 const selectedLocation = ref('all');
 const selectedPartner = ref('all');
+
+const handleExportClick = async () => {
+	try {
+		const { data } = await apollo.query({
+			query: userIdQuery
+		});
+
+		const userId = data?.my?.userAccount?.id;
+		if (!userId) {
+			console.error('No user ID available for export');
+			return;
+		}
+
+		const queryParams = new URLSearchParams({
+			iDisplayStart: '0',
+			iDisplayLength: props.totalLoans.toString(),
+			user_id: userId.toString()
+		});
+
+		window.location.href = `/portfolio/loans/export?${queryParams.toString()}`;
```

Comment:
The old export button downloaded a CSV, this one navigates away, is that an expected change?

Reply:
No, it's not supposed to take the user anywhere - just download a file.  When I tested it using https://www.kiva.org/portfolio/loans/export?${queryParams.toString()} it behaved fine for me.  But when it uses the local url, there's no file returned so it probably navigates to a blank page...

Reply:
When it goes into prod, it should use kiva.org/portfolio....

Reply:
Ah makes sense, it looks like we used `window.location.assign` before, does `window.location.href` similarly to trigger the download?

Reply:
Yeah it also triggers the download.

Uploading Screen Recording 2025-02-19 at 4.10.24 PM.mov…